readme.md

```diff
@@ -20,9 +20,18 @@ Use <History> </History> Inside the <Identity> tag and make sure to indent each
 
 We're writing for en_US. It really doesn't matter what you use but a tool like LanguageTool will help greatly with spellchecking. See 0 A.D.'s style guide here: https://trac.wildfiregames.com/wiki/EnglishStyleGuide
 
-Please use the US Customary system for measurements. I.E. feet not meters.
-  
-for consistency use B.C. and A.D. for the dates not BC and CE
+Here are some more that we've decided on:
+
+for consistency use BC and AD for the dates not BCE and CE
+
+Keep numbers simple. You can write One to Nine out or ones like Million but everything else write as numerals I.E. 12345 Instead of Twelve-Thousand Three hundred and Forty Five
+Likewise, all dates should be numerals. I.E. 100 AD
+Finally, we can use decimals instead of fractions ie. 1.5 instead 1 and 1/2 likewise use 2 to 3 instead of 2-3
+We use the US customary system for the measurements, and please do not abbreviate them.
+
+As for content, some parts of history can be gruesome but try to keep the writing fairly clean no please no gruesome tortures, or sacrifice rites.
+Also, History is some complicated so write in a clear, polite style that tries to see everything in context of the history and place.
+
```

Comment:
```suggestion
 Use the English version of any name, of there is one, for example: Plutarch (English) instead of Plutarchus (Latin) or Ploutarchos (Greek). If there is no English version, use Athenaeus(the Latin version) instead of Athenaios (Greek). If there is no Latin version, use the Greek version, like Xerxes (Greek) instead of the Persian version. For most names there is either a Greek or Latin version. If not, then use the name in the original language (can't think of an example). Names of People must always be capitalised. Same  for names of groups of people (like Helots, Hoplites)


```